src/docs/reference/modules/Contents/README.md

```diff
@@ -129,6 +129,136 @@ Well known properties include
     To log shapes call `@Orchard.ConsoleLog(Model.Content as object)` after calling `@await DisplayAsync(Model.Content)`
     This will allow the shape to execute, and populate the alternates for any child shapes.
 
+## Contents Module RESTful Web API
+
+The `OrchardCore.Contents` module provides RESTful API endpoints via [`ApiController`](https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore.Modules/OrchardCore.Contents/Controllers/ApiController.cs) featuring endpoints to manage _content items_. These endpoints allow for operations such as retrieving, creating, updating, and deleting single content item instances. Access to these endpoints requires authentication and appropriate user role permissions.
```

Comment:
In the meantime, there's no `ApiController` anymore but minimal API endpoints: https://github.com/OrchardCMS/OrchardCore/tree/main/src/OrchardCore.Modules/OrchardCore.Contents/Endpoints/Api. See https://github.com/OrchardCMS/OrchardCore/pull/15294.

Reply:
@Piedone what can I do? In the current version 1.8.2 we have ApiController, and I noticed that #15294 was merged into the main branch so is not available in 1.8.2, right?

Reply:
@Piedone and how can I use the Minimal API approach in the 1.8.2 version?

Reply:
@Piedone, is ApiController still supported in OC? Based on what I've read, I think Minimal APIs are not mature enough to be used in production (screenshot taken from [this page](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/apis?view=aspnetcore-8.0)): 

![image](https://github.com/OrchardCMS/OrchardCore/assets/19241483/59b98f99-d1b1-40e2-909a-c1d892a36882)

Reply:
As a consumer of the API, you don't have to do anything special, since it'll work exactly as it did before. However, the "ApiController" terminology is not correct anymore.

Reply:
Done [here](https://github.com/MarGraz/OrchardCore/commit/da34ccf51782c0e3dfdd991d45cf5807c30f296f)